etc/ReplayOfflineConfiguration.py

```diff
@@ -280,6 +280,29 @@
                  maxMemoryperCore=2000,
                  dataset_lifetime=14*24*3600,#lifetime for container rules. Default 14 days
                  versionOverride=expressVersionOverride)
+                 
+addExpressConfig(tier0Config, "ALCAPPS",
+                 scenario=ppScenario,
+                 data_tiers=["FEVT"],
+                 write_dqm=True,
```

Comment:
Maybe, for the meantime, this should be `False`?

Reply:
sorry, maybe I sent this too early... this is just about having the DQMIO output, not really about doing DQM, right?

Reply:
if you dont care about dqm output, yes.

Reply:
it's not really about the output, but the step itself. And also I'm not sure if we really dont need it, we'll need to hear from the PPS DQM experts